### PR TITLE
Add reporting capability

### PR DESCRIPTION
**Pull request by PippoRaimondi**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,15 +1,17 @@
 import createDir from './utils'
+import { createReport } from './reporter/reporter'
```
Comment (rafenden) [inline src/index.js]: Renaming `reporter/reporter.js` to `reporter/index.js` could make the import simpler, i.e.
```
import { createReport } from './reporter'
```

File: src/index.js [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-
+import TestStatus from './reporter/test-status'
 
 class WdioImage {
   constructor(browser, opts = undefined) {
     const options = opts || {}
 
     // Properties
     this.testName = 'Undefined test name'
+    this.testStatus = []
```
Comment (rafenden) [inline src/index.js]: Could be more intuitive to name this testStatus**es** as it's an array which holds multiple of them.

File: src/reporter/reporter.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+    if (arguments.length < 3)
+      throw new Error("Handlebars Helper equal needs 2 parameters")
+    if(lvalue != rvalue) {
+      return options.inverse(this)
+    } else {
+      return options.fn(this)
+    }
+  })
+
+  let templateFile = fs.readFileSync(path.resolve(__dirname, '../reporter/template.hbs'), 'utf8')
```
Comment (rafenden) [inline src/reporter/reporter.js]: Could be a `const`.

**Comment by PippoRaimondi** [comment]
fixes #3

**Pull request merged**